support-services/src/main/scala/com/gu/helpers/WebServiceHelper.scala

```diff
@@ -51,19 +65,32 @@ trait WebServiceHelper[Error <: Throwable] {
    */
   private def request[A](rb: Request.Builder)(implicit decoder: Decoder[A], errorDecoder: Decoder[Error], ctag: ClassTag[A]): Future[A] = {
     val req = wsPreExecute(rb).build()
-    SafeLogger.debug(s"Issuing request ${req.method} ${req.url}")
+    SafeLogger.info(s"Issuing request ${req.method} ${req.url}")
     for {
       response <- httpClient(req)
-    } yield {
-      val responseBody = response.body.string()
```

Comment:
body and string are both nullable according to okhttp3 docs

Reply:
sorry, body is nullable, but string might throw an IOException.  In any case we probably just want to blow up and retry with a RetryUnlimited